src/VisualStudio/Core/Def/Implementation/LanguageClient/RazorLanguageClient.cs

```diff
@@ -18,6 +18,7 @@
 
 namespace Microsoft.CodeAnalysis.ExternalAccess.Razor.Lsp
 {
+#if false
```

Comment:
Presumably this is just while you're testing?

Reply:
@dibarbet feels like we may not need this anymore.  will make determination before merging :)